source/_posts/en/practice/can-we-use-dom.md

```diff
@@ -1,78 +1,76 @@
 ---
-title: 我们可以操作 DOM 吗？
+title: can use operate dom?
 categories:
 - practice
 ---
 
-
-我们在使用 San 的时候，特别是刚刚使用不久的新人，且 MVVM 框架的经验不是那么丰富，我们还是更习惯于使用 jQuery 作为类库来操作页面的交互，于是很自然的写出了这样的代码。
+When we were using San, especially the newcomers who just used it, and the experience of the MVVM framework is not so rich, are still more accustomed to using jQuery as a class library to manipulate the interaction of the page, so it is natural to write such code.
```

Comment:
When san is introduced to some developers, especially those new to MVVM frameworks, may lead to jQuery style imperative implements like below